Automattic/vip-go-mu-plugins
# Refactor VIP Dashboard to not use React

## Description

The VIP Dashboard on `wp-admin` was originally intended to contain many features, but currently is simply a bunch of links and a form. For that reason, using React and a whole build system might be a bit overkill. 

This PR gets rid of React and the `gulp` build system, from which we were using very old versions. It replaces the pages for plain HTML, CSS and JS.

With this change, the functionality remains intact but we don't have to bother with CI or merging endless dependency upgrade PRs.

## Changelog Description

### Refactor VIP Dashboard to not use React

The VIP dashboard now has a way simpler implementation and keeps all its functionality.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Go to `wp-admin` > VIP Dashboard
3. Try to send a test ticket
